docs/sources/panels-visualizations/query-transform-data/transform-data/index.md

```diff
@@ -1216,6 +1218,8 @@ Select this option to transform the time series data frame from the long format
 | 2023-01-01 00:00:00 | 10     | 20     |
 | 2023-01-01 01:00:00 | 15     | 25     |
 
+> **Note:** This transformation is available in Grafana 7.5.10+ and Grafana 8.0.6+.
```

Comment:
@imatwawana I followed the instructions to generate this file and a couple unrelated lines popped in, is this okay? Did I do something wrong?

Reply:
i had this before and just manually reverted those changes